workspaces/redhat-argocd/plugins/argocd/package.json

```diff
@@ -22,10 +22,9 @@
   "scripts": {
     "build": "backstage-cli package build",
     "clean": "backstage-cli package clean",
-    "export-dynamic": "janus-cli package export-dynamic-plugin --in-place",
```

Comment:
Why do we require this change right now? Was this discussed in any forum with RHTAP folks that we need to remove the backwards compatibility for loading dynamic plugins from npm? AFAIK, RHTAP installer is not ready to adopt the new container image loading capabilities as of now. Removing this will break the installer since RHTAP is going to be using it in 1.4 release.

Reply:
As far as I know, the plugin is bundled via our showcase, but I converted it to a draft anyway so that it doesn't get merged by accident and we have time to discuss/share some info on Slack.

Reply:
correct, RHDH 1.4 bundles the rh-argocd plugin here https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd/package.json

in 1.5+ we plan to unbundle things but for now you get the dynamic plugin OOTB in a RHDH 1.4 install. Is the RHTAP installer not using the bundled plugin?

Reply:
The plan for RHTAP is to go out of the RHDH release schedule effectively being able to release as and when required. So after 1.3, the plan was to use RHTAP installer to release the latest version of the plugin. Now I am not sure if that will continue to work if we remove this support. Do we have some documentation for what it means to remove this support from dynamic plugins and what is the alternate way going forward?